weatherAplication/Model/DataModels/WeatherData.swift

```diff
@@ -1,9 +1,9 @@
 import UIKit
 
 struct WeatherData {
-    let hour: String
-    let temperature: Double
-    let pressure: Double
-    let windSpeed: Double
-    let weatherCode: Int
+    let hour: [String]
+    let temperature: [Double]
+    let pressure: [Double]
+    let windSpeed: [Double]
+    let weatherCode: [Int]
 }
```

Comment:
A czemu zmieniłeś tego structa w taki sposób? To rozwiązanie które było wcześniej moim zdaniem było lepsze, bo bardziej pasowało do typu danych używanych- mieliśmy dla danej godziny konkretne dane pogodowe. I również podczas używania od razu było widać że przekazujesz listę tych obiektów. Generalnie rzecz biorąc to struct nie powinien mieć w sobie **tylko** list, bo to się mija z samym jego założeniem